docs/getting-started/node-nvm.mdx

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
+---
+
+# NVM and Node.js
+
+## Uninstall Node.js(if you have it installed outside of nvm)
+
+Try to look for your current node.js file in `C:\Program Files` and uninstall it.
+
+
+## Installing the NVM 
+
+- Step 1: Head over to the [nvm-windows repository](https://github.com/coreybutler/nvm-windows#installation--upgrades) and click on **Download Now**
```

Comment:
There are no "Download Now" buttons at the location linked. In addition, it'd be a good idea to use ordered list (`1.`/`2.`) instead of unordered list (`-`).